PointOfDispensingApp/src/store/store.js

```diff
@@ -128,17 +128,11 @@ export default new Vuex.Store({
             state.screeningResponses.screeningComplete = false
         },
         patientRecordRetrieved(state, payload) {
+            console.log(payload);
             state.patientResource = payload.Patient;
-            if(state.patientHistory) {
-                state.patientHistory = payload.Immunization;
-                //TO DO:  We will also need to retrieve and populate the immunizationResource
-            } 
-            if(payload.Encounter) {
-                state.encounterResource = payload.Encounter;
-            }
-            if(payload.Appointment) {
-                state.appointmentResource = payload.Appointment;
-            }
```

Comment:
I had added these “if” statements to check that the resource is defined/exists before copying it into the store.  Should we keep those changes or not?

Reply:
That's a good point. I think it's better to check first. I'll add this back in